drivers/docker/config.go

```diff
@@ -189,7 +189,7 @@ var (
 			),
 			"image_delay": hclspec.NewDefault(
 				hclspec.NewAttr("image_delay", "string", false),
-				hclspec.NewLiteral("3m"),
+				hclspec.NewLiteral("\"3m\""),
```

Comment:
`"3m"` might read easier?

Reply:
That wouldn't quote the `3m` tho would it? - Could potentially do
```
`"3m"`
```
but I'm kinda against single line backticks

Reply:
@endocrimes yes, somehow github / me managed to do the ticks but forgot the quotes :) 

it's only a nit thing, I just personally find reading escape chars hard